src/services/navigateTo.ts

```diff
@@ -176,11 +176,8 @@ namespace ts.NavigateTo {
     function compareNavigateToItems(i1: RawNavigateToItem, i2: RawNavigateToItem): number {
         // TODO(cyrusn): get the gamut of comparisons that VS already uses here.
         // Right now we just sort by kind first, and then by name of the item.
-        // We first sort case insensitively.  So "Aaa" will come before "bar".

```

Comment:
`compareStringsCaseSensitive` already sorts correctly this way. It is possible this was written to address
the Node v0.10.0 issue with `String.prototype.localeCompare`, but it no longer seems necessary.

Reply:
After reverting `compareStringsCaseSensitive` to just use an ordinal relational comparison, it seems that `"aaa"` is once again greater than `"Aaa"`.